### PR TITLE
Third Try

### PR DESCRIPTION
**Pull request by Dorodomki**
## About The Pull Request

third try at make this code work lol

Adds a new jacket for the Marshals, Ranger, WO, Supply Specialist, Commisioned by pneumo/husky.

## Changelog
:cl:
add: Added new things
:cl:

image
-----------
image

### CODE REVIEW

**Comment by du-top** [comment]
Would you even be able to be _less_ descriptive?

File: code/modules/clothing/suits/storage.dm [hunk 1]
```diff
@@ -290,6 +290,16 @@
 	icon_open = "militaryjacket_soyfed_open"
 	icon_closed = "militaryjacket_soyfed"
 
+obj/item/clothing/suit/storage/toggle/miljacket_marshal
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/suits/storage.dm:293]:
```suggestion
/obj/item/clothing/suit/storage/toggle/miljacket_marshal
```

**Comment by SingingSpock** [comment]
Can you post an image of the current state of the sprites?

**Comment by Dorodomki** [comment]
> Can you post an image of the current state of the sprites?

they continue being the same sprites

File: code/modules/clothing/suits/storage.dm (hunk 1 above)
Comment (Dorodomki) [reply code/modules/clothing/suits/storage.dm:293]: thanks trilby i hate when 0 errors fails

**Comment by SingingSpock** [comment]
Then I continue opposing them. 1) Main sprites for any Marshal item shouldn't be _just_ blue. Blue and grey together are the Marshal colors, and while there are alt-sprites like that, it's not the case for main sprites. 2) As CBD pointed out, gold rank insignia are for WOs. If there's sprites that break that rule, then they should be changed, not more sprites that break that rule added.

**Comment by Dorodomki** [comment]
Captura de pantalla (1006)
----
Captura de pantalla (1004)

you can oppose to them all you want, the marshals use the colours mixed all the time in the "Formal Gear", Greatcoats, this statement you are doing is stupid and you just won't admit you are wrong wich is sad.

**Comment by SingingSpock** [comment]
Are either of those primary sprites for an item that's currently accessible? I know the lower one is specifically the WO greatcoat's alt-sprite

**Comment by Pawoverlord** [comment]
Honestly, 

From me going in to the topic and catching up on all the dialogue that was both on the Development General and on the PR, I'm pretty appalled by the behavior. It is _not_ our duty to _pay_ someone to _edit_ **their own** fluff items or contributions. If the item in question _does not fit_ the theme, lore, and rules of our server, (Especially while being combative and rude to those giving advice.) then it simply does not need to be pushed. 

Unless Skink or k5 want to discuss it further, I do not agree with this with this PR. Additionally, if this PR is to move forward it will need to comply with Sojourn standards for the Marshal department (As what it was intended for). And for the future, acting aggressive towards staff (even as proper staff) will almost absolutely foster a negative response from me, especially when the idea of 'requiring compensation for xyz' becomes involved; And I do not desire for this to become a future problem, either if it _does_ get pushed with the idea of 'you used my IP, therefore I demand compensation'.

So! To avoid **all** of that drama, I will officially put my stamp of denial here until (at least one of) the condition is met, that being, of being adjusted **or** absolute 100% permission to be adjusted by the Sojourn Spiriting Team is given (Permitting that any of our spriters are willing to touch this and make adjustments.)

**Comment by cdb-is-not-good** [comment]
In light of the above comment I'm marking this as a draft for now and marking it D.N,M.

**Comment by Dorodomki** [comment]
> So! To avoid **all** of that drama, I will officially put my stamp of denial here until (at least one of) the condition(s) is met, that being, of being adjusted **or** absolute 100% permission to be adjusted by the Sojourn Spiriting Team is given (Permitting that any of our spriters are willing to touch this and make adjustments.)

- check discord dms

**Comment by Pawoverlord** [comment]
After speaking with several people on this topic namely getting a better understanding of the situation at large, I want to present an idea to see about getting this merge moving again. 

The jacket's blue palette in its current state is completely fine. If anyone has an issue with it needing to be more gray or more blue, they can speak with me directly. As for that gold bit; I wanted to touch base with you, Dorodomki, on the thought train of keeping the gold trim for the WO, and having an additional jacket with the gold trim swapped for a gray or black trim. 

Keep in mind this is only asking as a hypothetical. If you're not open to the idea, I may have an alternative solution that can try to maintain a peaceful ground between contributors, staff, and playerbase.

**Comment by Trilbyspaceclone** [comment]
> After speaking with several people on this topic namely getting a better understanding of the situation at large, I want to present an idea to see about getting this merge moving again.
>

Seems you nolonger are blocking its merge, if im incorrect just let me know

**Comment by Pawoverlord** [comment]
I'd hold off just a moment before the merge, allow me until tomorrow to finish tying up some loose ends and resolving the headache.

**Comment by cdb-is-not-good** [comment]
Unmarking till then, i suppose. Let's probably just leave this as a draft till otherwise stated.

**Comment by Pawoverlord** [comment]
image

Leaving this here so that relevant information is where it's needed. 
TL:DR Commissioner quit Sojourn because of what happened here. And artist is pulling their work away from Sojourn because the person it was made for is no longer among the community.

**Pull request merged**